src/pybind/mgr/volumes/fs/nfs.py

```diff
@@ -529,6 +530,9 @@ def create_export(self, fs_name, cluster_id, pseudo_path, read_only, path):
             if not self.check_fs(fs_name):
                 return -errno.ENOENT, "", f"filesystem {fs_name} not found"
 
+            if not isabs(pseudo_path):
+                return -errno.EINVAL, "", f"pseudo path {pseudo_path} is not an absolute path"
```

Comment:
Next, please also check that `pseudo_path` is not `/` (also invalid!). Write a test too.

Reply:
Also done.